### PR TITLE
docs: add instructions to build app with OSTk:

### PR DESCRIPTION
**Pull request by vishwa2710**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,97 @@ Once the container is running, access [http://localhost:8888/lab](http://localho
 - [Use Cases](./docs/Use%20Cases.md)
 - [Frequently Asked Questions](./docs/FAQ.md)
 
+## Using OSTk in your applications
+
+To help jump-start using OSTk, here is a simple docker image that you can use to create your applications.
```
Comment (lucas-bremond) [inline README.md:59]:
```suggestion
To help jump-start using OSTk, here is a simple Docker image that you can use to create your applications.
```

File: README.md [hunk 2]
```diff
@@ -54,6 +54,97 @@ Once the container is running, access [http://localhost:8888/lab](http://localho
...
+
+```toml
+[project]
+name = "my-app"
+requires-python = ">=3.11"
+
+dynamic = ["version"]
+
+dependencies = [
+    # Public libraries
```
Comment (lucas-bremond) [inline README.md:126]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -54,6 +54,97 @@ Once the container is running, access [http://localhost:8888/lab](http://localho
...
+    "ipython~=8.24",
+]
+```
+
+### Build and Run
+
+You can then build the Docker image via:
+```
+docker build . -t my_OSTk_app
+docker run -it --rm my_OSTk_app
```
Comment (lucas-bremond) [inline README.md:142]:
```suggestion
docker build . -t my-app
docker run -it --rm my-app
```

to be in sync w/ the above

File: README.md [hunk 4]
```diff
@@ -54,6 +54,97 @@ Once the container is running, access [http://localhost:8888/lab](http://localho
...
+
+FROM mcr.microsoft.com/vscode/devcontainers/python:${PYTHON_BASE_VERSION}
+
+ARG OSTK_DATA_LOCAL_CACHE="/var/cache/open-space-toolkit-data"
+
+## Install OSTk data install dependencies
+
+RUN apt-get update \
+    && apt-get install -y git-lfs \
+    && rm -rf /var/lib/apt/lists/*
```
Comment (lucas-bremond) [inline README.md:76]:
```suggestion
RUN apt-get update. && \
    apt-get install -y git-lfs && \
    rm -rf /var/lib/apt/lists/*
```

File: README.md [hunk 5]
```diff
@@ -54,6 +54,97 @@ Once the container is running, access [http://localhost:8888/lab](http://localho
...
+
+You can then build the Docker image via:
+```
+docker build . -t my_OSTk_app
+docker run -it --rm my_OSTk_app
+```
+
+Enjoy!
+
+
```
Comment (lucas-bremond) [inline README.md:147]:
```suggestion
```

**Review by lucas-bremond — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -54,6 +54,95 @@ Once the container is running, access [http://localhost:8888/lab](http://localho
...
+FROM mcr.microsoft.com/vscode/devcontainers/python:${PYTHON_BASE_VERSION}
+
+ARG OSTK_DATA_LOCAL_CACHE="/var/cache/open-space-toolkit-data"
+
+## Install OSTk data install dependencies
+
+RUN apt-get update \
+    && apt-get install -y git-lfs \
+    && rm -rf /var/lib/apt/lists/*
```
Comment (vishwa2710) [inline README.md:76]:
```suggestion
RUN apt-get update && \
    apt-get install -y git-lfs && \
    rm -rf /var/lib/apt/lists/*
```

**Pull request merged**